Clarified description of Version attribute in Trac. (#19776)
django/django: docs/internals/contributing/triaging-tickets.txt

Version
-------

> The *version* attribute indicates the version in which the bug was reproduced.
> During triage, it is helpful to update this field to the earliest supported
> version that reproduces the issue, but there is no need to make further updates
> when that version goes out of support. Avoid resetting the version to "dev"
> merely to indicate the issue still exists. Instead, reference the commit hash
> you tested in a comment.

Reviewer: How about:
```suggestion
The *version* attribute indicates the earliest version in which the bug was    
reproduced. During triage, this field can be updated but there is no need to   
make further updates when that version goes out of support. The field should   
not be reset to "dev" to show the issue still exists: instead, the tested      
commit hash can be noted in a comment.                                         
```
